python-project-template/README.md.jinja

```diff
@@ -37,7 +37,7 @@ Once you have created a new environment, you can install this project for local
 development using the following commands:
 
 ```
->> pip install -e .'[dev]'
+>> ./.setup_dev.sh
 >> pre-commit install
```

Comment:
Can remove this line, as `pre-commit install` is run in `/.setup_dev.sh`.

Reply:
Removed this, and altered the note about pre-commit install below.